server/index.js

```diff
@@ -2,16 +2,17 @@ require("dotenv").config();
 const express = require('express')
 const app = express();
 const axios = require('axios');
-const port = process.env.SERVER_PORT || 8080;
-const dbMethods = require('./database/index.js')
+const port = process.env.SERVER_PORT
 const { dashboard } = require('./routes');
-const { createQuiz } = require ('./routes')
+const { create } = require('./routes/index.js')
+const bodyParser = require('body-parser')
 
 // =============================================
 //                Middleware
 // =============================================
 app.use(express.json());
-
+app.use(bodyParser.json());
```

Comment:
Was there issues using express.json?

Reply:
Wasn't 100% sure if express.json was working when debugging the initial connection, was just an attempt to test if it work, but it worked after. I can test without it bodyparser and remove it if express.json works as intended with logic.